version/src/lib.rs

```diff
@@ -35,7 +35,7 @@ pub mod error {
     use super::UnsupportedVersion;
 
     /// Versioning errors
-    #[derive(Debug, Clone, FromVariant, IntoSchema)]
```

Comment:
What's the rationale?

Reply:
Also in a few places you removed the `Clone` implementation from Error variants. Why is that?

Reply:
The intention of the removal of these `Clone` implementations is just a revert against what I added [earlier](https://github.com/hyperledger/iroha/pull/1867) but later found unnecessary.
Indeed, I did not touch `UnsupportedVersion` and `RawVersioned` at that time. Fixed

Reply:
The intention of the removal of these `Clone` implementations is just a revert against what I added [earlier](https://github.com/hyperledger/iroha/pull/1867) but later found unnecessary.
Indeed, I did not touch `UnsupportedVersion` and `RawVersioned` at that time. Fixed